Governance.md

```diff
@@ -0,0 +1,78 @@
+# Governance of the rules_scala
+
+These bazel rules for scala are under the bazelbuild org but governed by a
+contributors from the community.
+
+## Goals of this document
+
+Clarify:
+0. values of the project.
+1. how to get a PR merged
+2. how long you should need to wait for a review
+3. how to resolve conflicts in project direction
+4. how to make larger and disruptive changes
+
+## Values of rules_scala
```

Comment:
It would be great to have a value around accessibility and maintainability. Perhaps accessibility is a synonym for usability.

The general idea behind accessibility being that the rules can be adopted and used by many throughout the Scala ecosystem. When users move to adopt the rules, we are welcoming and helpful to them doing so. If their use case requires small, non-breaking changes to the rule set, we can help them make them. If their use case requires broader changes, but can be put behind a flag, then we can help there as well.

Maintainability is more about keeping up with Bazel best practices, updating the codebase as Bazel evolves, and preventing the codebase from devolving into some organically grown spaghetti. If we keep the codebase easier to work on, it is more likely to be worked on.

Reply:
I'm good with this. Not sure about exact phrasing but I'm good with it. I think they need to be in the bottom.